README.md

```diff
@@ -384,6 +392,12 @@ See `help(g.featurize)` for more options
       g.umap(kind='nodes', X=['col_1', ..., 'col_n'], y=['label', ..., 'other_targets'], ...)
     ```
 
+* `umap(engine="...")` supports multiple implementations. It defaults to using the GPU-accelerated `engine="cuml"` when a GPU is available, resulting in orders-of-magnitude speedups, and falls back to CPU processing via `engine="umap"`.:
+
```

Comment:
should the `engine="umap_learn"` above? I see `UMAPEngineConcrete = Literal["cuml", "umap_learn"]` are the options.